src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Config.cs

```diff
@@ -11,5 +11,6 @@ internal static class Config
     public static string DotNetDirectory { get; } = Environment.GetEnvironmentVariable("DOTNET_DIR") ?? "./.dotnet";
     public static string DotNetTarballPath { get; } = Environment.GetEnvironmentVariable(DotNetTarballPathEnv) ?? string.Empty;
     public const string DotNetTarballPathEnv = "DOTNET_TARBALL_PATH";
+    public static bool ExcludeOnlineTests { get; } = Environment.GetEnvironmentVariable("EXCLUDE_ONLINE_TESTS") != null;
```

Comment:
```suggestion
    public static bool ExcludeOnlineTests { get; } = Convert.ToBoolean(Environment.GetEnvironmentVariable("EXCLUDE_ONLINE_TESTS"));
```

I think it's more intuitive to avoid the issue with the environment variable being `EXCLUDE_ONLINE_TESTS=false` causing them to be excluded.

Reply:
Fair suggestion.  I think I like avoiding exceptions for this scenario though so how about - bool.TryParse(Environment.GetEnvironmentVariable("EXCLUDE_ONLINE_TESTS"), out bool excludeOnlineTests) ? excludeOnlineTests : false;